### DISCUSSION TITLE
Is there any chance to integrate Rectangle with Raycast?

### DISCUSSION THREAD

**Discussion by francisconoriega** [rxhanson/Rectangle#924] [Ideas]
I really like Rectangle, and prefer typing commands over mouse gestures, and I'm terrible at remembering shortcuts, so something like Raycast really works for me. I'd love to be able to do `Cmd+Space`, and then just type `Almost Maximize` to be able to trigger that action in Rectangle. Is there any API to trigger the commands with the terminal instead of with shortcuts?

**Comment by CanRau** [comment]
A little late to the party, tho wanted to add for anyone interested that there's https://www.raycast.com/extensions/window-management
I'm in no way affiliated, just switching more stuff to raycast 🤓

**Comment by rxhanson** [comment]
Forgot to update this discussion, but a scriptable interface was added a while back, via URLs:

https://github.com/rxhanson/Rectangle?tab=readme-ov-file#execute-an-action-by-url
